format/format.go

```diff
@@ -103,7 +103,7 @@ func (f *Format) format(path string) error {
 }
 
 func write(path string, contents []byte) error {
-	f, err := ioutil.TempFile(filepath.Split(path))
+	f, err := ioutil.TempFile(filepath.Dir(path), filepath.Base(path))
```

Comment:
This line is the same as formatter.go (L372) before modifying  #1192 
https://github.com/swaggo/swag/pull/1192/files